queries/default.sparql

```diff
@@ -67,6 +66,14 @@ WHERE {
     }.
     ?item wdt:P31 ?not
   }
+  # Remove various comman false postives that are any subclass of
```

Comment:
```suggestion
  # Remove various common false positives that are any subclass of
```

Reply:
@swedebugia why did you mark this as resolved? The typos are still in the diff...

Reply:
Sorry, did not know the meaning of the button I clicked.